core/dbt/lib.py

```diff
@@ -1,10 +1,16 @@
-# TODO: this file is one big TODO
 import os
 from dbt.exceptions import RuntimeException
 from dbt import flags
-from collections import namedtuple
+from dataclasses import dataclass
 
-RuntimeArgs = namedtuple("RuntimeArgs", "project_dir profiles_dir single_threaded profile target")
+
```

Comment:
@drewbanin I know this is technically a change for 1.3, but there's no concern about this _not_ working prior to 1.3, correct? 

I'm up for testing if necessary, but I don't know how to run dbt from a local branch as a different version than the latest and would need a little assistance.

Reply:
just confirmed that this logic works in dbt v.1.2-latest! I think a `namedtuple` basically works like a class.... no clue why we went that route instead using a dataclass in the first place. We should be a-ok here